.changeset/neat-singers-bathe.md

```diff
@@ -0,0 +1,9 @@
+---
+"@fuel-ts/contract": minor
+"@fuel-ts/example-contract": minor
+"forc-bin": minor
```

Comment:
We need to remove `forc-bin` from here otherwise, the CI will change the version and break.

Can you try to find a why for changesets to ignore this package? Maybe we can open another issue.

Reply:
done!